src/main/java/org/skriptlang/skript/bukkit/potion/elements/expressions/ExprSecPotionEffect.java

```diff
@@ -0,0 +1,218 @@
+package org.skriptlang.skript.bukkit.potion.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SectionExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.util.Timespan.TimePeriod;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import ch.njol.util.Math2;
+import org.bukkit.event.HandlerList;
+import org.bukkit.potion.PotionEffect;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.potion.util.PotionUtils;
+import org.skriptlang.skript.bukkit.potion.util.SkriptPotionEffect;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffectType;
+import org.skriptlang.skript.registration.SyntaxInfo;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("Potion Effect")
```

Comment:
```suggestion
@Name("New Potion Effect")
```
I think this fits it better but up to you